### DISCUSSION TITLE
pre tag triggers Livewire: Multiple root elements detected.

### DISCUSSION THREAD

**Discussion by patrykszady** [livewire/livewire#5512] [7. Bugs (archived)]
Hi, title says it all, no matter what I try, whenever I have a `<pre>` tag in my component I receive the `Livewire: Multiple root elements detected. ` warning.

**Comment by joshhanley** [comment]
@patrykszady thanks for reporting! I'm unable to replicate the issue. I just tested with a component that has the below html and it all worked happily for me. Could you share your code which reproduces the issue with as little code as possible so we can investigate it?

Thanks!

```blade
<div>
    <pre>
        Test
    </pre>
</div>
```
